### DISCUSSION TITLE
get the full-file-path using github API

### DISCUSSION THREAD

**Discussion by AFeTest** [community/community#49285] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

For each file in the pull request, I want to get the full file path.
I found a way to only get the file name, without path.
There is any option to get the File full path ?
Thanks!

**Comment by james-ecd** [comment]
Yes, you can use the GitHub REST API to get the full file path for each file in a pull request. Here's an example API call that you can use:

```
GET /repos/:owner/:repo/pulls/:pull_number/files
```

This API call returns a list of all the files that were modified, added, or deleted in the pull request, along with their full paths. Each file is represented by a JSON object with the following properties:

- filename: The name of the file, including its extension.
- status: The status of the file in the pull request (modified, added, or removed).
- additions: The number of lines added to the file in the pull request.
- deletions: The number of lines deleted from the file in the pull request.
- changes: The total number of lines changed in the file in the pull request.
- blob_url: The URL to view the file's contents.
- raw_url: The URL to download the file's contents.
- contents_url: The API URL to get the contents of the file.

To get the full path of each file, you can simply concatenate the file's path in the repository with its filename. For example, if the path of a file is src/app/index.js and its filename is index.js, then the full path of the file is src/app/index.js.
